### DISCUSSION TITLE
友链朋友圈beta版(已适配volantis主题)

### DISCUSSION THREAD

**Discussion by Zfour** [volantis-x/hexo-theme-volantis#595] [Ideas]
预览：https://zfe.space/friendcircle/
教程（其实是寻找小白鼠）：https://zfe.space/post/friend-link-circle.html
有任何问题可在这里留言。
预览效果：
![image](https://user-images.githubusercontent.com/19563906/109308132-9110b080-787c-11eb-8bc9-00e02232ffc1.png)

**Comment by xaoxuu** [comment]
厉害啊 👍 主题这边需要做什么不
